sourcemash/frontend/templates/category.tpl

```diff
@@ -19,6 +19,8 @@
           <span class="badge"><a href="#/categories/<%=item.get('category_2')%>"><%=item.get('category_2')%></a></span>
           <h4><a href="<%= '#/items/' + item.get('id') %>"><%= item.get('title') %></a></h4>
         </span>
+        <span class="waves-effect btn-floating red"><i class="mdi-action-thumb-down downvote"></i></span>
```

Comment:
Niceness of the frontend is an issue for another day. We'll end up doing a small design overhaul sometime in the near future to get everything looking nice, but it's not necessary for now since it's highly subject to change.